docs/src/index.md

```diff
@@ -15,6 +15,35 @@ using ACEfit
 using PythonCall
 ```
 
+## MLJ solvers
+
+To use [MLJ](https://github.com/alan-turing-institute/MLJ.jl) solvers you need to load MLJ in addition to ACEfit
+
+```julia
+using ACEfit
+using MLJ
```

Comment:
Conceptually, does it make sense to recommend that MLJ is loaded first?

Reply:
Loading order does not matter, so I would leave it as it is.

Reply:
The example in `ext/ACEfit_MLJLinearModels_ext.jl` loads MLJ first - can we make them consistent?